hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/CreateHoodieTableCommand.scala

```diff
@@ -110,6 +112,13 @@ object CreateHoodieTableCommand {
     }
   }
 
+  def validateMOR(tableConfig: HoodieTableConfig): Unit = {
+    if (tableConfig.getTableType.name.equals(DataSourceWriteOptions.MOR_TABLE_TYPE_OPT_VAL)
+      && StringUtils.isNullOrEmpty(tableConfig.getPreCombineField)) {
```

Comment:
Should we validate based on the table config, wondering how we have a table config if the table does not exist yet?

Reply:
I guess this is called only after init table. so table config will be available

Reply:
Yes, you are right ~